Src/Core/WeaponController.cs

```diff
@@ -55,44 +51,32 @@ void Update()
                 swingAudioSource.Play();
             }
 
-            if(!swingBack) attackAngle = 3f * (weaponRenderer.flipY ? -1 : 1);
-            else attackAngle = -3f * (weaponRenderer.flipY ? -1 : 1);
+            attackAngle = swingBack ? -3f * (weaponRenderer.flipY ? -1 : 1) : 3f * (weaponRenderer.flipY ? -1 : 1);
```

Comment:
You could change this to ``attackAngle = (swingBack ? -3f : 3f) * (weaponRenderer.flipY ? -1 : 1)`` to make it even better